gtsfm/frontend/homography_verifier/homography_verifier_base.py

```diff
@@ -0,0 +1,74 @@
+"""Base class for the V (verification) stage of the frontend.
```

Comment:
This has the same description as `verifier/verifier_base.py`. Maybe specify that this is applying verification wrt to a Homography matrix